jobs:
  open-issue:
    runs-on: self-hosted
    steps:
    - continue-on-error: true
      uses: actions/checkout@v4
    - continue-on-error: true
      env:
        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
      name: Open issue
      run: "gh issue create --title \"$GITHUB_WORKFLOW #$GITHUB_RUN_NUMBER failed\"\
        \ \\\n                --label Bug \\\n                --body \"See [$GITHUB_WORKFLOW\
        \ #$GITHUB_RUN_NUMBER](https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID).\"\
        \n"
name: Reusable - Open issue on workflow failure
on:
  repository_dispatch:
    types: trigger-ga___reusable-open-issue-on-failure.yml
